Dockerfile.indexserver

```diff
@@ -1,9 +1,8 @@
-FROM alpine:3.16.2
+FROM alpine:3.16
```

Comment:
we intentionally specify the patch version to prevent docker image caching leading to random patch versions used.

I'll update to 3.16.4 which is the same (right now) as 3.16